lib/db/filters/StudyRequestFiltersSql.js

```diff
@@ -56,9 +56,6 @@ function getTextSearchFilter(column) {
  */
 function getStudyRequestFiltersSearch(column, query) {
   const filtersSearch = [];
-  if (column === null || column === 'ASSIGNED_TO') {
-    filtersSearch.push(getTextSearchFilter('"searchAssignedTo"'));
```

Comment:
this does technically mean we don't need those columns anymore, but we can clean them up later (e.g. if we need to squeeze a bit more performance out of Track Requests at some point).